presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -797,12 +801,14 @@ The Iceberg connector can delete data in one or more entire partitions from tabl
 
 .. note::
 
-    Columns in the filter must all be identity transformed partition columns of the target table.
-
     Filtered columns only support comparison operators, such as EQUALS, LESS THAN, or LESS THAN EQUALS.
 
     Deletes must only occur on the latest snapshot.
 
+    For V1 tables, the connector can only delete data in one or more entire
+    partitions. Columns in the filter must all be identity transformed partition
```

Comment:
No change, just noting that "Columns in the filter must all be..." is clearly phrased. Nice work!